build/config/win/BUILD.gn

```diff
@@ -9,9 +9,6 @@ import("//build/config/compiler/compiler.gni")
 import("//build/config/sanitizers/sanitizers.gni")
 import("//build/config/win/control_flow_guard.gni")
 import("//build/config/win/visual_studio_version.gni")
-if (!is_starboard) {
```

Comment:
Instead of trying to move these changes to //starboard/build, what you most likely want to try to do is switch these conditionals. For example, this could be switched to `if (!use_cobalt_customizations)`, as could the conditional on L5 of build/config/win/visual_studio_version.gni below.

Reply:
Done.